observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -91,15 +91,21 @@ The Debug Exporter prints traces and metrics to the standard output.
     exporters:
       debug:
         verbosity: detailed # <1>
+        sampling_initial: 5 # <2>
+        sampling_thereafter: 200 # <3>
+	use_internal_logger: true # <4>
     service:
       pipelines:
         traces:
-          exporters: [logging]
+          exporters: [debug]
         metrics:
-          exporters: [logging]
+          exporters: [debug]
 # ...
 ----
 <1> Verbosity of the debug export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+<2> "number of messages initially logged each second" The default value is `2` seconds.
```

Comment:
```adoc
<2> The number of messages initially logged each second. The default value is `2`.
```

Is the default value two seconds or two messages?

Reply:
The parameter sets the number of messages logged not the value for the time period. 
sampling_initial (default = 2): number of messages initially logged each second.

Reply:
Thank you both.